nautobot/ipam/tests/test_models.py

```diff
@@ -1301,6 +1301,20 @@ def test_get_or_create(self):
                 str(err.exception),
             )
 
+    def test_get_or_create_address_kwarg(self):
+        status = Status.objects.get(name="Active")
+        namespace = Namespace.objects.create(name="Test IPAddress get_or_create with address kwarg")
+        Prefix.objects.create(prefix="10.0.0.0/24", namespace=namespace, status=status)
+        _, created = IPAddress.objects.get_or_create(
+            address="10.0.0.40/32", namespace=namespace, defaults={"status": status}
+        )
+        self.assertTrue(created)
```

Comment:
Would be good to assert that `host` and `mask_length` are set correctly on the created IP?